Example App throws errors when inserting data into DB

**Describe the bug**
The example app throws multiple errors like `Caused by: org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "consumption_points_pkey"`, when new data is received via Kafka.
Investigate the cause and fix it.

**To reproduce**
Describe the steps to reproduce the behavior:
1. Checkout main
2. Start example app
3. Use HVD data need and start simulator
4. Submit some state and random data
5. Possibly have to repeat with a new connectionId
6. See example app logs for error

**Expected behavior**
No errors are thrown, the data is inserted successfully into the DB.

--- a/examples/example-app/src/main/java/energy/eddie/examples/exampleapp/kafka/KafkaListener.java
+++ b/examples/example-app/src/main/java/energy/eddie/examples/exampleapp/kafka/KafkaListener.java
@@ -6,6 +6,7 @@ import energy.eddie.cim.v0_82.cmd.ConsentMarketDocument;
 import energy.eddie.cim.v0_82.cmd.MktActivityRecordComplexType;
 import energy.eddie.cim.v0_82.cmd.PermissionComplexType;
 import energy.eddie.cim.v0_82.vhd.PointComplexType;
+import energy.eddie.cim.v0_82.vhd.SeriesPeriodComplexType;
 import energy.eddie.cim.v0_82.vhd.TimeSeriesComplexType;
 import energy.eddie.examples.exampleapp.Env;
 import energy.eddie.examples.exampleapp.kafka.serdes.ConsentMarketDocumentSerde;
@@ -125,22 +126,23 @@ public class KafkaListener implements Runnable {
                     .executeAndReturnGeneratedKeys("id")
                     .mapTo(Integer.class)
                     .first();
+            int order = 0;
+            for (SeriesPeriodComplexType period : seriesPeriods.getSeriesPeriods()) {
+                for (PointComplexType point : period.getPointList().getPoints()) {
+                    BigDecimal consumption = point.getEnergyQuantityQuantity();
+                    String type = point.getEnergyQuantityQuality().value();
 
-            seriesPeriods.getSeriesPeriods().forEach(period -> {
-                PointComplexType first = period.getPointList().getPoints().getFirst();
-                BigDecimal consumption = first.getEnergyQuantityQuantity();
-                Integer order = Integer.valueOf(first.getPosition());
-                String type = first.getEnergyQuantityQuality().value();
-
-                h.createUpdate("""
-                                INSERT INTO consumption_points(consumption_record_id, ord, consumption, metering_type) VALUES (?, ?, ?, ?)
-                                """)
-                        .bind(0, id)
-                        .bind(1, order)
-                        .bind(2, consumption)
-                        .bind(3, type)
-                        .execute();
-            });
+                    h.createUpdate("""
+                                           INSERT INTO consumption_points(consumption_record_id, ord, consumption, metering_type) VALUES (?, ?, ?, ?)
+                                           """)
+                            .bind(0, id)
+                            .bind(1, order)
+                            .bind(2, consumption)
+                            .bind(3, type)
+                            .execute();
+                    order++;
+                }
+            }
             return null;
         });
     }

